client/webserver/site/src/js/markets.js

```diff
@@ -812,13 +812,12 @@ export default class MarketsPage extends BasePage {
     if (order.targetID) {
       const targetOrder = this.metaOrders[order.targetID]
       if (!targetOrder) return
-      Doc.hide(Doc.tmplElement(targetOrder.row, 'cancelStatus'))
       if (note.subject === 'cancel') {
-        targetOrder.order.status = Order.StatusCanceled
+        Doc.tmplElement(targetOrder.row, 'cancelStatus').textContent = 'canceled'
       } else if (note.subject === 'revoke') {
         targetOrder.order.status = Order.StatusRevoked
+        updateUserOrderRow(targetOrder.row, targetOrder)
       }
-      updateUserOrderRow(targetOrder.row, targetOrder)
```

Comment:
Note that _before_ I broke the cancel updates, it looked like the following: https://github.com/decred/dcrdex/blob/2f9b30839c1dcc7ca84be467abbc4f13b50023b9/client/webserver/site/src/js/markets.js#L810-L822

I still don't understand how that (hiding `cancelStatus` *and* updating its `textContent`) worked for cancel notes, but going back to just updating `textContent` instead of trying to update the underlying metaOrder's status restores proper operation afaict.